index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Microverse's Little Survey</title>
+</head>
+<body>
+  
+  <h1 id="title">How Do You Know Microverse?</h1>
+  
+  <p id="description">This form help us to know how do you know about Microverse</p>
```

Comment:
I would suggest utilizing tags such as h2 or h3 for text of this nature, which serves as a subtitle. It is important to bear in mind that the p tag is typically employed for lengthier passages of text.